deploy-tool/config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"github.com/pelletier/go-toml"
+)
+
+type DeployConfig struct {
+	DNS     DNSConfig
+	ACME    ACMEConfig
+	Storage StorageConfig
+}
+
+type DNSConfig struct {
+	Provider         string
+	CloudflareConfig *CloudflareConfig `toml:"Cloudflare"`
+}
+
+type CloudflareConfig struct {
+	APIKey string `toml:"api_key"`
+}
+
+type ACMEConfig struct {
+	Email               string
+	CA                  string
```

Comment:
`CA` is a little ambiguous. Going by the value in the sample config file, this is the ACME API endpoint, so I think the field should be named accordingly.